vortex-array/src/array/bool/mod.rs

```diff
@@ -48,6 +55,24 @@ impl BoolArray {
         )
     }
 
+    pub fn into_boolean_builder(self) -> (BooleanBufferBuilder, usize) {
```

Comment:
does this need to be `pub`? don't particularly like the fn signature (you have to read the body to know what the usize in the returned tuple is)

Reply:
This is the same method as `into_maybe_null_slice` just that this is a bitset and arrow doesn't preserve the bit offset